src/Liip/RMT/Command/BaseCommand.php

```diff
@@ -34,7 +35,11 @@ public function run(InputInterface $input, OutputInterface $output)
         // Store the input and output for easier usage
         $this->input = $input;
         $this->output = $output;
-        $this->output->setDialogHelper($this->getHelperSet()->get('dialog'));
+        $dialogHelper = class_exists('Symfony\Component\Console\Helper\QuestionHelper')
```

Comment:
The test suite is pretty good! But, it's never testing dialog interaction, it's only testing calling the commands with input parameters, so a manual test need to be done. I will do it right now